Makefile

```diff
@@ -54,6 +54,9 @@ init: node_modules ;
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
+	find source -name '*.json' | xargs -n 1 jsonlint -q
```

Comment:
@vyp nice catch; just pushed a fix. I don't think it changes anything but it's always nice to be correct :)